src/System.Net.HttpListener/tests/SimpleHttpTests.cs

```diff
@@ -159,5 +165,66 @@ public async Task UnknownHeaders_Success(int numHeaders)
                 }
             }
         }
+
+        [Fact]
+        public void ListenerRestart_BeginGetContext_Success()
+        {
+            using (HttpListenerFactory factory = new HttpListenerFactory())
+            {
+                HttpListener  listener = factory.GetListener();
```

Comment:
Should `listener` be used or closed here?

Reply:
no, I don't think so. Factory will create listener and call Start() on it. So it should be ready to go.